x/ibc/04-channel/types/packet.go

```diff
@@ -63,6 +64,9 @@ func (p Packet) GetDestChannel() string { return p.DestinationChannel }
 // GetData implements PacketI interface
 func (p Packet) GetData() []byte { return p.Data }
 
+// GetTimeoutEpoch implements PacketI interface
+func (p Packet) GetTimeoutEpoch() uint64 { return p.TimeoutEpoch }
```

Comment:
Hmm, why this two-function strategy? Better to have a `GetTimeoutHeight() Height` function instead, no?

Reply:
Currently this would introduce an import cycle between `02-client/exported` and `04-client/exported`